### ISSUE TITLE
multi-line descriptions break the generator

### ISSUE THREAD

**Issue by NeverwinterMoon** [maticzav/swift-graphql#36]
**Describe the bug**
Apollo GraphQL allows something like this:
```
    """
    Establish permissions the user might have for the bike.

    Used for FW variant "abc"
    """
    bikePermission(
      id: String!
    ): BikeConnectResult
```

The generator outputs the above as:
```swift
/// Establish permissions the user might have for the bike.

    Used for FW variant "abc"

    func bikePermission<Type>(id: String, selection: Selection<Type, Unions.BikeConnectResult?>) throws -> Type {
```

**Expected behavior**
Multi-line docs are supported and do not create an uncompilable code

**Comment by Amzd** [comment]
Is multi-line also supported for deprecation messages in apollo?

**Comment by NeverwinterMoon** [comment]
@Amzd I have no idea, really. But why do you ask?

**Comment by maticzav** [comment]
This has been fixed in #54. Feel free to comment if the issue persists.
